doc/bitcoin-upstream-merge.md

```diff
@@ -0,0 +1,82 @@
+Bitcoin Upstream Merge Guide
+====================================
+DigiByte was originally forked from the Bitcoin & Litecoin UTXO codebase. Since 2014 many unique changes have been made to DGB that separate it from the latest BTC codebase. However, there is a tremendous amount of improvements continually being made by talented open-source blockchain developers from around the world to the Bitcoin codebase that DigiByte can quite often benefit from.
```

Comment:
Grammar update: "However, there are a tremendous amount of improvements continuously being made by ..."

Reply:
Change made